nautobot/docs/release-notes/version-2.2.md

```diff
@@ -66,6 +66,36 @@ The CSV import functionality for all models has been changed from a synchronous
 Model CharFields' `max_length` attributes have been standardized globally to have at least 255 characters except where a shorter `max_length` is explicitly justified.
 
 <!-- towncrier release notes start -->
+## vv2.2.2 (2024-04-18)
```

Comment:
```suggestion
## v2.2.2 (2024-04-18)
```

I'm guessing you're doing `towncrier build --version v2.2.2` instead of `towncrier build --version 2.2.2` :)

Reply:
Ahh I see nice catch thank you!